### DISCUSSION TITLE
Change volume location on QNAP Installation

### DISCUSSION THREAD

**Discussion by alexkiddddd** [immich-app/immich#6248] [Q&A]
Hi there, I am running immich on a QNAP NAS with container station, I have installed it with the default docker-compose.yml on the github and everything is working smoothly.

I have noticed that immich volumes are on container station folder and I cannot access them to backup, is it possible to change the volume location to another folder without breaking immich?

Thanks!

**Comment by Tonylolo** [comment]
Hi, 
i tested on Portainer on Qnap too, and I have the same problem, how change UPLOAD LOCATION for a directory outside the docker ?
